### ISSUE TITLE
Updating firmware page is a bit hard to follow

### ISSUE THREAD

**Issue by chris1seto** [WildernessLabs/Documentation#45]
"Download the latest Netduino firmware files from the downloads page. You’ll need the Tinybooter, ER_CONFIG, ER_FLASH files for your specific Netduino model."

What package are these files inside? I see the Netduino3Wifi_4_3_2_2 dir, but the files are the wrong extension for MFDeploy

**Comment by bryancostanich** [comment]
Note to self: files are .s19, which is fine for the Netduino windows updater and the Mac Deploy, but MFDeploy expects .hex.

**Comment by bryancostanich** [comment]
Further note to self: need to remove the MFDeploy stuff and replace with the Secret Labs Netduino updater, since it's better than MFDeploy; it'll unbrick a device, or deploy to a device that doesn't have tinybooter because it uses DFU, rather than MFUSB.
